src/ui/forum.html

```diff
@@ -7,8 +7,9 @@
 <head>
 
     <meta charset="utf-8" />
-    <title>Button Men Online &mdash; Forum</title>
+    <title>Button Men Online</title>
```

Comment:
Ah, this is the same, so i bet you did it on purpose and are going to generate these headers dynamically - i'll keep reading.  :>)